edx-olive-oc/edx-platform
# CAM2-30 Updates translations

Updates some translations Sign In/Registration, and the incomplete profile alerts.

**JIRA tickets**:  CAM2-30, OC-4870

**Discussions**: See tickets for link to translations spreadsheet.

**Screenshots**: 
*Sign In (Hebrew)*
![sign in he](https://user-images.githubusercontent.com/7556571/44019196-3c7b716c-9f1d-11e8-9df0-c063b1bcf376.png)

*Sign In (Arabic)*
![sign in ar](https://user-images.githubusercontent.com/7556571/44019339-969bb97c-9f1d-11e8-99cb-cdd75b8f93e1.png)

*Register (Arabic)*
![register ar](https://user-images.githubusercontent.com/7556571/44019374-b8294c76-9f1d-11e8-93fd-22ac7b8ac306.png)

*Register (Hebrew)*
![register he](https://user-images.githubusercontent.com/7556571/44019359-ac87a80e-9f1d-11e8-9f71-071b10cd9142.png)

**Sandbox URL**: TBD - sandbox is being provisioned.

**Testing instructions**:

1. Work through the changed translations on the spreadsheet, and ensure that they display correctly in the LMS.
1. Verify the process used to update these translations.
   Note that because we're on a modified Ginkgo branch, most of these strings are not available in Transifex, and so we have to do them manually. Process is:
    1. Update string in `django.po` / `djangojs.po` files for each affected language.
    1. Run `./manage.py lms compilemessages` to compile the `.po` to `.mo` files.
    1. Run `./manage.py lms compilejsi18n` to compile the `djangojs.mo` files into the `djangojs.js` files.

**Author Notes & Concerns**
1. python test failures should be unrelated to these translation changes -- will be fixed as part of CAM2-69 / OC-5041

**Reviewers**
